util.rb

```diff
@@ -0,0 +1,60 @@
+require 'sinatra/base'
+require_relative 'environment'
+
+# IAM - a resource usage metric collection and reporting system
+class Util
+  def self.max_value(data)
+    # find the max Value
+    highest = 0 if data.empty?
+    highest = data[0][:value].abs
+    data.each do |item|
+      highest = item[:value].abs if item[:value].abs > highest
+    end
+    highest
+  end
+
+  def self.min_value(data)
+    # find the lowest Value
+    lowest = 0 if data.empty?
+    lowest = data[0][:value].abs
+    data.each do |item|
+      lowest = item[:value].abs if item[:value].abs < lowest
+    end
+    lowest
+  end
+
+  def self.avg_mean_value(data)
+    # find the average(Mean) of the Values
+    average = 0
+    count = 0
+    average = 0 if data.empty?
+    data.each do |item|
+      average += item[:value].abs
+      count += 1
+    end
+    average /= count
+  end
+
+  def self.active_bool(data)
+    # insure that Active is either True or False
+    data.each do |item|
+      if item[:active] == true || item[:active] == false
+        # do nothing
+      else
+        return false
+      end
+    end
+    true
```

Comment:
In this method you explicitly return true but in the following method you don't.  Is there any particular reason?  Do both work?  If they both work, is one approach better than the other?